hudi-cli/src/main/java/org/apache/hudi/cli/commands/CompactionCommand.java

```diff
@@ -246,7 +247,7 @@ public String compact(
       Option<String> firstPendingInstant =
           client.reloadActiveTimeline().filterCompletedAndCompactionInstants()
               .filter(instant -> instant.getAction().equals(HoodieTimeline.COMPACTION_ACTION)).firstInstant()
-              .map(HoodieInstant::getTimestamp);
+              .map(HoodieInstant::requestedTime);
```

Comment:
Do we already reach consensus to rename to `requestedTime` ? In incremental query, we have unified the query range as `startTime` and `completionTime`, should we unify the naming concention of all these, kind of a mess.

Reply:
Forget it, I thought it again and `requestTime` does not really conflict with incremental query `startTime`, the former is for instant request time(start time), the later is for query range start time(which is actually a completion time).

They are two kind of things technically.

Reply:
Yes, finalized requestedTime